src/managed/OpenLiveWriter.BlogClient/Clients/GoogleBloggerv3Client.cs

```diff
@@ -88,7 +88,7 @@ private static BlogPost ConvertToBlogPost(Post post)
                 Permalink = post.Url,
                 Contents = post.Content,
                 DatePublished = post.Published.Value,
-                Keywords = string.Join(new string(LabelDelimiter,1), post.Labels ?? new List<string>())
+                Categories = post.Labels.Select(x => new BlogPostCategory(x)).ToArray()
```

Comment:
@kathweaver If its not captured already, opening posts from the remote blog server would be good to add to the test plan. You can do this via  **File** > **Open recent post** > **Your Blog Name**. Then you can browse through and select any post, and Writer will download the post from the blog server and open it as a local draft.